### PR TITLE
Add account setup & billing doc

### PR DESCRIPTION
**Pull request by etcadinfinitum**

### CODE REVIEW

File: _docs/cloud_account_setup.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Account Creation & Setup
+
+Begin the registration process for each provider's free tier at the following
+pages:
+
+* [Azure for Students](https://azure.microsoft.com/en-us/free/free-account-students-faq/); click `Activate` to begin the registration process
+* [Azure](https://azure.microsoft.com/en-us/free/); click `Start free`
+* [Google Cloud Platform](https://cloud.google.com/free/): click `Get started for free`
+* [AWS Educate](https://www.awseducate.com/registration#APP_TYPE); select `Student` and fill out the web form to apply for student benefits. Make sure to use your `.edu` email address.
+* [AWS](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc); click `Create A Free Account
```
Comment (etcadinfinitum) [inline _docs/cloud_account_setup.md]: Closing backtick
```suggestion
* [AWS](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc); click `Create A Free Account`
```

File: _docs/cloud_account_setup.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+| Microsoft Azure | Azure has a [Free Tier](https://azure.microsoft.com/en-us/free/); many common services are free to use and **new customers get a $200 platform credit**. | Azure offers [a special platform credit for students](https://azure.microsoft.com/en-us/free/free-account-students-faq/); a $100 platform credit is included and **registration for Azure for Students does not require a credit card on file**. | For enterprise systems and corporate customers, [Azure may provide advantages and disadvantages](https://www.informationweek.com/cloud/infrastructure-as-a-service/hidden-cloud-costs-aws-azure-management-costs-compared/a/d-id/1298029) depending on the existing system configuration, but for students, the differency is likely not noticable. |
```
Comment (anishprasad01) [inline _docs/cloud_account_setup.md]:
```suggestion
| Microsoft Azure | Azure has a [Free Tier](https://azure.microsoft.com/en-us/free/); many common services are free to use and **new customers get a $200 platform credit**. | Azure offers [a special platform credit for students](https://azure.microsoft.com/en-us/free/free-account-students-faq/); a $100 platform credit is included and **registration for Azure for Students does not require a credit card on file**. | For enterprise systems and corporate customers, [Azure may provide advantages and disadvantages](https://www.informationweek.com/cloud/infrastructure-as-a-service/hidden-cloud-costs-aws-azure-management-costs-compared/a/d-id/1298029) depending on the existing system configuration, but for students, the difference is likely not noticable. |
```

Comment (anishprasad01) [reply _docs/cloud_account_setup.md]: Just a spelling error. (Differency to Difference)

**Review by anishprasad01 — APPROVED** [review]
Looks great! Verified that all links work and go to the correct destinations. Other than one tiny spelling mistake, it looks good to go.

**Pull request merged**